### PR TITLE
feature/35: edit card

### PR DESCRIPTION
**Pull request by Gerasimov94**

### CODE REVIEW

File: src/screens/Rooms/components/RoomEditRoom.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+						/>
+					</View>
+					<Text style={styles.addImageContainerTitle}>Add Image</Text>
+				</View>
+				<ScrollView
+					contentContainerStyle={styles.imagesGallery}
+					horizontal
+				>
+					{this.props.room.images.map((item: string, index: number) => (
+						<RoomEditCoomImageContainer
```
Comment (CodeAnnihilator) [inline src/screens/Rooms/components/RoomEditRoom.tsx]: Coom?

File: src/common/components/reduxForm/reduxFormInput.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, {PureComponent} from 'react';
+import {TextInput} from 'react-native';
+
+export default class ReduxFormInput extends PureComponent<any> {
```
Comment (CodeAnnihilator) [inline src/common/components/reduxForm/reduxFormInput.tsx]: add interface

File: src/screens/Rooms/components/RoomEditRoom.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+import {ScrollView, View, Text} from 'react-native';
+import {Field} from 'redux-form';
+
+import styles from '../styles/RoomEditRoom.styles';
+
+import ImagePicker from '../../../common/components/ImagePickerWrapper';
+import ReduxFormInput from '../../../common/components/reduxForm/reduxFormInput';
+import RoomEditRoomImageContainer from '../containers/RoomEditRoomImageContainer';
+
+export default class RoomEditRoom extends React.Component<any> {
```
Comment (CodeAnnihilator) [inline src/screens/Rooms/components/RoomEditRoom.tsx]: add interface

File: src/screens/Rooms/components/RoomEditRoom.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+							style={styles.editRoomItemContent}
+						/>
+					</View>
+					<Text style={styles.addImageContainerTitle}>Add Image</Text>
+				</View>
+				<ScrollView
+					contentContainerStyle={styles.imagesGallery}
+					horizontal
+				>
+					{this.props.room.images.map((item: string, index: number) => (
```
Comment (CodeAnnihilator) [inline src/screens/Rooms/components/RoomEditRoom.tsx]: destr props

**Review by CodeAnnihilator — CHANGES_REQUESTED** [review]
please add comments under each conversation like done/fixed/discussed

File: src/screens/Rooms/components/RoomEditRoom.tsx (hunk 4 above)
Comment (Gerasimov94) [reply src/screens/Rooms/components/RoomEditRoom.tsx]: i'm use a props once, why i need to desctr props?

File: src/common/components/reduxForm/reduxFormInput.tsx (hunk 2 above)
Comment (Gerasimov94) [reply src/common/components/reduxForm/reduxFormInput.tsx]: fixed

File: src/screens/Rooms/components/RoomEditRoom.tsx (hunk 3 above)
Comment (Gerasimov94) [reply src/screens/Rooms/components/RoomEditRoom.tsx]: fixed

File: src/screens/Rooms/components/RoomEditRoom.tsx (hunk 4 above)
Comment (Gerasimov94) [reply src/screens/Rooms/components/RoomEditRoom.tsx]: but ok, i'm make it

File: src/screens/Rooms/components/RoomEditRoom.tsx (hunk 1 above)
Comment (Gerasimov94) [reply src/screens/Rooms/components/RoomEditRoom.tsx]: fixed

**Review by CodeAnnihilator — APPROVED** [review]

**Pull request merged**